### PR TITLE
[DSLX] Add stdlib path flag to `interpreter_main`.

### PR DESCRIPTION
**Pull request by cdleary**
This is helpful for outside-the-repository environments where you may want to point at some install prefix for the standard library.

### CODE REVIEW

File: xls/dslx/run_routines/run_routines.h [hunk 1]
```diff
@@ -33,8 +33,8 @@
 #include "absl/status/statusor.h"
 #include "absl/time/time.h"
 #include "absl/types/span.h"
+#include "re2/re2.h"
 #include "xls/dslx/bytecode/bytecode_interpreter_options.h"
-#include "xls/dslx/default_dslx_stdlib_path.h"
```
Comment (hzeller) [inline xls/dslx/run_routines/run_routines.h:37]: If  this header is now removed, we probably don't need to link `default_dslx_stdlib_path` anymore.

```
bant dwyu //xls/dslx/run_routines:run_routines
```

Comment (cdleary) [reply xls/dslx/run_routines/run_routines.h:37]: Done -- were you planning to make this part of presubmit?

Comment (hzeller) [reply xls/dslx/run_routines/run_routines.h:37]: One of the next times I am build-gardener ...

**Review by hzeller — APPROVED** [review]
LGTM

**Pull request merged**